backend/serve.go

```diff
@@ -291,6 +321,20 @@ func Manage(pluginID string, serveOpts ServeOpts) error {
 		}
 	}()
 
+	if serveOpts.HandlerMiddlewares == nil {
+		serveOpts.HandlerMiddlewares = make([]HandlerMiddleware, 0)
+	}
+
+	middlewares := []HandlerMiddleware{
+		newContextualLoggerMiddleware(),
+		NewTracingMiddleware(tracing.DefaultTracer()),
+		NewMetricsMiddleware(prometheus.DefaultRegisterer, "grafana", false),
+		NewLoggerMiddleware(Logger, nil),
+		newHeaderMiddleware(),
+		NewErrorSourceMiddleware(),
+	}
+	serveOpts.HandlerMiddlewares = append(middlewares, serveOpts.HandlerMiddlewares...)
```

Comment:
Register the default SDK middlewares applicable to any plugin using the SDK. Grafana can then reuse this and providing an empty set of middlewares.